webpack/assets/javascripts/react_app/components/HostDetails/Parameters/styles.scss

```diff
@@ -0,0 +1,5 @@
+.pf-c-accordion {
```

Comment:
Should we namespace it a bit? or do we really want to apply the rule on all the accordions?